libraries/botframework-connector/botframework/connector/auth/jwt_token_validation.py

```diff
@@ -61,13 +68,21 @@ class JwtTokenValidation:
         channel_service: str,
         channel_id: str,
         service_url: str = None,
+        auth_configuration: AuthenticationConfiguration = None,
     ) -> ClaimsIdentity:
         if not auth_header:
             raise ValueError("argument auth_header is null")
 
-        using_emulator = EmulatorValidation.is_token_from_emulator(auth_header)
+        if SkillValidation.is_skill_token(auth_header):
+            return await SkillValidation.authenticate_channel_token(
+                auth_header,
+                credentials,
+                channel_service,  # TODO
```

Comment:
what;s this todo?

Reply:
i.e. what is pending here?

Reply:
I should remove that comment, I was considering porting the ChannelProvider interface but I remembered I had that functionality in a helper class (JS do it that way)